docs/src/app/components/pages/components/popover.jsx

```diff
@@ -87,6 +87,16 @@ let PopoverPage = React.createClass({
       <ComponentDoc
         name="Popover"
         componentInfo={componentInfo}>
+        
+        <Paper style = {{marginBottom: '22px'}}>
+          <CodeBlock>
+          {
+            '//Import statement:\nimport Popover from \'material-ui/lib/popover\';\n\n' +
```

Comment:
Alright, I just checked with another component.
Could you fix the travis task ? I will merge then. Thanks.

IMHO, we should have a component to render this. I will work on it.